schema/workbook.json

```diff
@@ -1,2 +1,754 @@
+                }
+            },
+            "required": [
+                "namespace",
+                "metric",
+                "aggregation"
+            ]
+        },
+        "metricFilters": {
+            "$id": "#/definitions/metricFilters",
+            "type": "object",
+            "properties": {
+                "key": {
+                    "type": "string",
+                    "description": "the dimension to filter on"
+                },
+                "operator": {
+                    "type": "integer",
+                    "description": "the equality operation of the filter"
+                },
+                "values": {
+                    "type": "array",
+                    "description": "the values that the filters should match, OR'd together",
+                    "items": [
+                        {
+                            "type": "string"
+                        }
+                    ]
+                },
+                "valueParam": {
+                    "type": "string",
+                    "description": "if applied, the parameter chosen as the filter value"
+                }
+            },
+            "required": [
+                "key",
+                "operator",
+                "values"
+            ]
+        },
+        "linkItem": {
+            "$id": "#/definitions/linkItem",
+            "type": "object",
+            "properties": {
+                "linkTarget": {
+                    "type": "string",
+                    "description": "What the link should go to"
+                },
+                "linkLabel": {
+                    "type": "string",
+                    "description": "What the link should say",
+                    "tags": [
+                        "localizable"
+                    ]
+                },
+                "preText": {
+                    "type": "string",
+                    "description": "What text comes before the link",
+                    "tags": [
+                        "localizable"
+                    ]
+                },
+                "postText": {
+                    "type": "string",
+                    "description": "What text comes after the link",
+                    "tags": [
+                        "localizable"
+                    ]
+                },
+                "style": {
+                    "type": "string",
+                    "examples": [
+                        "link",
+                        "primary",
+                        "secondary"
+                    ]
+                },
+                "linkIsContextBlade": {
+                    "type": "boolean",
+                    "description": "Open up the link in a context blade"
+                }
+            }
+        },
+        "parameterItem": {
+            "$id": "#/definitions/parameterItem",
+            "type": "object",
+            "properties": {
```

Comment:
i think parameter item also has a thing for dropdowns of what value to display for the "defaults" item that could be localized?

Reply:
I wasn't able to find this option. If you are able to find it can you point me to it?